assets/xml/objects/object_sekihig.xml

```diff
@@ -1,16 +1,19 @@
 ﻿<Root>
+    <!-- Assets for a suns song grave with a goron symbol on it -->
     <File Name="object_sekihig" Segment="6">
-        <DList Name="gSunSongGoronSymbolBaseDL" Offset="0x340" />
-        <DList Name="gSunSongGoronSymbolDL" Offset="0x4A8" />
-        <DList Name="gSekihigGraveBaseDL" Offset="0x4B8" />
-        <DList Name="gGraveGoronSymbolDL" Offset="0x5C8" />
-        <Texture Name="gSekihigTLUT" OutName="sekihig_tlut" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
-        <!-- <Blob Name="object_sekihig_Blob_000600" Size="0x60" Offset="0x600" /> -->
-        <Texture Name="gSekihigGraniteTex" OutName="sekihig_granite" Format="rgba16" Width="32" Height="32" Offset="0x660" />
-        <Texture Name="gSekihigMarbleTex" OutName="sekihig_marble" Format="rgba16" Width="32" Height="64" Offset="0xE60" />
-        <Texture Name="gSekihigGoronSymbolTex" OutName="sekihig_goron_symbol" Format="ci4" Width="64" Height="64" Offset="0x1E60" />
-        <Texture Name="gSekihigStoneInscriptionTex" OutName="sekihig_stone_inscription" Format="ia4" Width="128" Height="16" Offset="0x2660" />
-        <Texture Name="gSekihigStaffNotesTex" OutName="sekihig_staff_notes" Format="ia4" Width="128" Height="32" Offset="0x2A60" />
-        <Collision Name="gSekihigCol" Offset="0x33D8" />
+        <DList Name="gSunsSongGraveGoronSymbolsBaseDL" Offset="0x340" />
+        <DList Name="gSunsSongGraveGoronSymbolsDL" Offset="0x4A8" />
+        <DList Name="gSunsSongGraveGoronStoneDL" Offset="0x4B8" />
+        <DList Name="gSunsSongGraveGoronDL" Offset="0x5C8" />
+        <Texture Name="gSunsSongGraveGoronTLUT" OutName="sun_song_grave_goron_tlut" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
+        <Texture Name="gSunsSongGraveGoronKokiriTLUT" OutName="suns_song_grave_goron_kokiri_tlut" Format="rgba16" Width="4" Height="4" Offset="0x600" />
+        <Texture Name="gSunsSongGraveGoronUnkownTLUT" OutName="suns_song_grave_goron_unkown_tlut" Format="rgba16" Width="4" Height="4" Offset="0x620" />
```

Comment:
```suggestion
        <Texture Name="gSunsSongGraveGoronUnknownTLUT" OutName="suns_song_grave_goron_unknown_tlut" Format="rgba16" Width="4" Height="4" Offset="0x620" />
```

Reply:
One of these days I will spell `Unknown` right.